packages/blocks/src/__internal__/utils/selection.ts

```diff
@@ -139,10 +139,19 @@ function focusRichTextByModel(
   position: SelectionPosition,
   model: BaseBlockModel
 ) {
-  const element = getBlockElementByModel(model);
-  const editableContainer = element?.querySelector('[contenteditable]');
-  if (editableContainer) {
-    focusRichText(position, editableContainer);
+   // FIXME: Then it will turn the input into the div
+  if (model.flavour === 'group') {
```

Comment:
I think this looks too ad-hoc and violate the semantic of this function. Can we explicitly detect when is this case in `handleKeyUp`?

Reply:
ok, that's a good idea, I will fix it.